### PR TITLE
Add security-updates-only configuration for renovate

### PR DESCRIPTION
**Pull request by internalsystemerror**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | no
| BC Break      | no
| New Feature   | yes
| RFC           | no
| QA            | yes

### Description

As discussed at the [September 12th 2022 meeting](https://github.com/laminas/technical-steering-committee/blob/main/meetings/minutes/2022-09-12-TSC-Minutes.md#security-only-packages-and-dependency-updates) of the TSC, we intend to roll out renovate for "security-updates-only" repositories of Laminas.

This PR:
- adds the shared configuration for use in those repositories
- fixes a minor documentation typo (missing closing **)
- adds documentation for the new configuration
- updates renovates owner from Whitesource to Mend as noted at https://github.com/whitesource

### CODE REVIEW

**Comment by Ocramius** [comment]
@internalsystemerror wrong branch deleted? 🤔

**Comment by internalsystemerror** [comment]
ahhhhhhhhh bugger ... i already left now too :/ ~~dnt suppose u still haz?~~ 🤣 actually i shud still have it locally so i'll repush tomorrow, sorry

**Comment by internalsystemerror** [comment]
restored.

File: renovate-config-security-updates-only.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "description": "Configuration for repositories marked 'security-updates-only' in the Laminas organisation",
+    "extends": [
+        "local>laminas/.github:renovate-config",
+        ":maintainLockFilesDisabled"
+    ],
+    "packageRules": [
+        {"matchPackagePatterns": ["*"], "enabled": false},
+        {"matchPackageNames": ["php"], "enabled": true, "rangeStrategy": "widen"}
```
Comment (Ocramius) [inline renovate-config-security-updates-only.json:10]: Should we create draft MRs to widen the PHP version also here?

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: renovate-config-security-updates-only.json (hunk 1 above)
Comment (internalsystemerror) [reply renovate-config-security-updates-only.json:10]: This extends the default, so should inherit the specifics from that, meaning we could remove the "rangeStrategy": "widen" part and only need to disable all packages except for PHP. I will update.

**Comment by internalsystemerror** [comment]
It no longer seems to be tracking the branch on my fork, should I close this and reopen?

**Comment by Ocramius** [comment]
If you think it has an effect: sure!

**Pull request closed without merging**